.github/workflows/perform-post-comment.yml

```diff
@@ -0,0 +1,52 @@
+name: Quarkiverse Perform post Initial Comment to PR
+
+on:
+  # Called in the pr comment workflow
+  workflow_call:
+    inputs:
+      report-config-file:
+        description: |
+          relative path to file report-config.yml
+        default: .github/report-config.yml
+        type: string
+permissions:
+  pull-requests: write
+
+jobs:
+  comment:
+    name: (Re)Create initial comment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Lookup report config file
+        id: lookup
+        run: |
+          present=false
+          if [[ -f ${CONFIG_FILE} ]]
+          then
+            present=true
+          fi
+          echo "config-present=${present}" >> ${GITHUB_OUTPUT}
+        env:
+          CONFIG_FILE: ${{ inputs.report-config-file }}
+      - name: Check report config is not empty
+        id: check
+        if: ${{ steps.lookup.outputs.config-present == 'true' }}
+        uses: mikefarah/yq@v4
```

Comment:
`yq` is already present in the `ubuntu-latest` distribution

Reply:
Restructured.